server/routes/profile.go

```diff
@@ -52,12 +52,19 @@ func CheckDatastore(w http.ResponseWriter, r *http.Request) {
 }
 
 func GetProfile(w http.ResponseWriter, r *http.Request) {
-	ctx := context.Background()
+	if r.Method != http.MethodGet {
```

Comment:
I'm pretty sure we've set up the router in a way that it responds with 405 (Method Not Allowed) automatically in this case